### ISSUE TITLE
Not enough value to unpack error when making chips

### ISSUE THREAD

**Issue by magtanggol03** [azavea/raster-vision#767]
Hi, 

I am currently trying to train a household detection model on 1970 tiled satellite images with labels from OpenStreetMaps. 

The code runs into errors when doing "making chips for scene: train-x", where it takes 5 hours to generate 500+ out of the 1970, then suddenly gets a ValueError. Not sure why, and the image this happens on is a bit random as i tried running this multiple times and this error would appear at different chips.

This is the error that I'm getting:

```
...
2019-04-12 17:03:46:rastervision.backend.tf_object_detection: INFO - Creating TFRecord
2019-04-12 17:03:46:rastervision.task.task: INFO - Making train chips for scene: train-653
2019-04-12 17:04:30:rastervision.backend.tf_object_detection: INFO - Creating TFRecord
2019-04-12 17:04:31:rastervision.task.task: INFO - Making train chips for scene: train-654
2019-04-12 17:05:04:rastervision.backend.tf_object_detection: INFO - Creating TFRecord
2019-04-12 17:05:05:rastervision.task.task: INFO - Making train chips for scene: train-655
Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/opt/src/rastervision/__main__.py", line 17, in <module>
    rv.main()
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/opt/src/rastervision/cli/main.py", line 159, in run
    dry_run=dry_run)
  File "/opt/src/rastervision/runner/experiment_runner.py", line 187, in run
    self._run_experiment(command_dag)
  File "/opt/src/rastervision/runner/local_experiment_runner.py", line 35, in _run_experiment
    run_commands(tmp_dir)
  File "/opt/src/rastervision/runner/local_experiment_runner.py", line 29, in run_commands
    command.run(tmp_dir)
  File "/opt/src/rastervision/command/chip_command.py", line 20, in run
    self.augmentors, tmp_dir)
  File "/opt/src/rastervision/task/task.py", line 126, in make_chips
    train_scenes, TRAIN, augment=True)
  File "/opt/src/rastervision/task/task.py", line 122, in _process_scenes
    return [_process_scene(scene, type_, augment) for scene in scenes]
  File "/opt/src/rastervision/task/task.py", line 122, in <listcomp>
    return [_process_scene(scene, type_, augment) for scene in scenes]
  File "/opt/src/rastervision/task/task.py", line 112, in _process_scene
    data.shuffle()
  File "/opt/src/rastervision/core/training_data.py", line 36, in shuffle
    self.chips, self.windows, self.labels = zip(*chip_windows_labels)
ValueError: not enough values to unpack (expected 3, got 0)
```

Btw im using this version: `quay.io/azavea/raster-vision:gpu-0.8`

Thanks so much!

**Comment by lewfish** [comment]
I would upgrade to `quay.io/azavea/raster-vision:gpu-0.9` (we just released `0.9.0rc1`). This has the ability to parallelize chipping and prediction, btw, using the `--splits n` option to `rastervision run`. This also contains a bug fix for your problem.

**Comment by magtanggol03** [comment]
This works, thanks!
